telescope/selector.py

```diff
@@ -85,24 +85,36 @@ def __init__(self):
     self.duration = None
     self.metrics = None
     self.ip_translation_spec = None
-    self.client_providers = None
-    self.sites = None
+    self.client_providers = [None]
+    self.client_countries = [None]
+    self.sites = [None]
 
   def split(self):
     """Splits a MultiSelector into an equivalent list of Selectors."""
     selectors = []
-    selector_product = itertools.product(
-        self.start_times, self.client_providers, self.sites, self.metrics)
-    for start_time, client_provider, site, metric in selector_product:
+    selector_product = itertools.product(self.start_times,
+        self.client_providers, self.client_countries, self.sites, self.metrics)
+    for start_time, client_provider, client_country, site, \
+            metric in selector_product:
         selector = Selector()
         selector.ip_translation_spec = self.ip_translation_spec
         selector.duration = self.duration
         selector.start_time = start_time
-        selector.client_provider = client_provider
-        selector.site = site
         selector.metric = metric
         selector.mlab_project = _metric_to_project[metric]
         selectors.append(selector)
+
+        selector.client_provider = client_provider
+        selector.client_country = client_country
+        selector.site = site
+        
+        if selector.client_provider != None:
+            selector.client_provider = selector.client_provider.lower()
+        if selector.client_country != None:
+            selector.client_country = selector.client_country.lower()
```

Comment:
We should add `client_country` to `Selector.__init__` so that it's obvious what attributes a `Selector` object should have.